torch/csrc/xpu/Stream.cpp

```diff
@@ -22,20 +22,27 @@ static PyObject* THXPStream_pynew(
   int64_t stream_id = 0;
   int64_t device_index = 0;
   int64_t device_type = 0;
+  uint64_t queue_ptr = 0;
 
   // NOLINTNEXTLINE(modernize-avoid-c-arrays,cppcoreguidelines-avoid-c-arrays)
   constexpr const char* kwlist[] = {
-      "priority", "stream_id", "device_index", "device_type", nullptr};
+      "priority",
+      "stream_id",
+      "device_index",
+      "device_type",
+      "queue_ptr",
+      nullptr};
   if (!PyArg_ParseTupleAndKeywords(
           args,
           kwargs,
-          "|iLLL",
+          "|iLLLK",
```

Comment:
Why did you add this to the base Stream constructor? We don't actually use it here?

Reply:
I originally wanted to align with the constructor of CUDA's base Stream. It is okay to remove it. And add it back if really needed in future.